modules/serverless-functions-func-yaml-fields.adoc

```diff
@@ -0,0 +1,137 @@
+
+== envs
+
+The `envs` field enables you to set environment variables to be available to your function at runtime. You can set an environment variable in several different ways:
+
+. Directly from a value.
+. From a value assigned to a local environment variable.
+. From a key-value pair stored in a secret or config map.
+. You can also import all key-value pairs stored in a secret or config map, with keys used as names of the created environment variables.
+
+This examples demonstrates the different ways to set an environment variable:
+
+[source,yaml]
+----
+name: test
+namespace: ""
+runtime: go
+...
+envs:
+- name: EXAMPLE1 <1>
+  value: value
+- name: EXAMPLE2 <2>
+  value: '{{ env:LOCAL_ENV_VALUE }}'
+- name: EXAMPLE3 <3>
+  value: '{{ secret:mysecret:key }}'
+- name: EXAMPLE4 <4>
+  value: '{{ configMap:myconfigmap:key }}'
+- value: '{{ secret:mysecret2 }}' <5>
+- value: '{{ configMap:myconfigmap2 }}' <6>
+----
+<1> An environment variable set directly from a value.
+<2> An environment variable set from a value assigned to a local environment variable.
+<3> An environment variable assigned from a key-value pair stored in a secret.
+<4> An environment variable assigned from a key-value pair stored in a config map.
+<5> A set of environment variables imported from key-value pairs of a secret.
+<6> A set of environment variables imported from key-value pairs of a config map.
+
+== volumes
+
+The `volumes` field enables you to mount secrets and config maps as a volume accessible to the function at the specified path. For example:
+
+[source,yaml]
+----
+name: test
+namespace: ""
+runtime: go
+...
+volumes:
+- secret: mysecret <1>
+  path: /workspace/secret
+- configMap: myconfigmap <2>
+  path: /workspace/configmap
+----
+<1> The `mysecret` secret is mounted as a volume residing at `/workspace/secret`.
+<2> The `myconfigmap` config map is mounted as a volume residing at `/workspace/configmap`.
+
+== options
+
+The `options` field enables you to modify Knative Service properties for the deployed function, such as autoscaling. If these options are not set, the default ones are used.
+
+These options are available:
+
+* `scale`
+** `min`: The minimum number of replicas. Must be a non-negative integer. The default is 0.
+** `max`: The maximum number of replicas. Must be a non-negative integer. The default is 0, which means no limit.
+** `metric`: Defines which metric type is watched by the Autoscaler. Can be set to `concurrency`, which is the default, or `rps`.
+** `target`: Recommendation for when to scale up based on the number of concurrently incoming requests. The `target` option can be a float value greater than 0.01. The default is 100, unless the `options.resources.limits.concurrency` is set, in which case `target` defaults to its value.
+** `utilization`: Percentage of concurrent requests utilization allowed before scaling up. Can be a float value between 1 and 100. The default is 70.
```

Comment:
Same as previous comment

```suggestion
** `utilization`: Percentage of concurrent requests utilization allowed before scaling up. It can be a float value between 1 and 100. The default is 70.
```